src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+const tree = document.querySelector('.tree');
+
+for (const listItem of list) {
+  if (listItem.children.length > 0) {
+    const span = document.createElement('span');
+
+    listItem.prepend(span);
+    span.append(listItem.childNodes[1]);
+  }
+}
+
+tree.addEventListener('click', event => {
+  if (event.target.tagName !== 'SPAN') {
+    return;
+  }
+
+  const element = event.target;
+  const elementDisplay = element.nextSibling.style.display;
+
+  if (elementDisplay !== 'none') {
+    element.nextSibling.style.display = 'none';
+  }
+
+  if (elementDisplay === 'none') {
+    element.nextSibling.style.display = 'block';
+  }
+});
```

Comment:
instead of changing display style try to use[ .hidden prop](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden)
hint: just change the hidden prop to the opposite value (elem.hidden = !elem.hidden)